css/main.css

```diff
@@ -1,29 +1,161 @@
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+img {
+  max-width: 100%;
+}
+ul,
+ol {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
 body {
   font-family: "Lato", sans-serif;
 }
 .page_header {
+  width: 100%;
+  position: absolute;
+  color: #fff;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 15px;
+  padding-right: 15px;
+  padding-top: 54px;
+}
+.hero-section {
   background: linear-gradient(
-    90deg,
-    rgba(213, 32, 71, 0.8) 0%,
-    rgba(238, 61, 27, 0.8) 100%
-  );
+      90deg,
+      rgba(213, 32, 71, 0.8) 0%,
+      rgba(238, 61, 27, 0.8) 100%
+    ),
+    url(/images/Rectangle_3.jpg);
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
```suggestion
  background-size: cover;
```

Reply:
I agree with you, after "bs: cover" it will be redundant.